huangjoyce3/MealMatch
# [Backend] Update schema for `delivery_request`, `delivery`

Trello: [https://trello.com/c/jVTxPlS0]( https://trello.com/c/jVTxPlS0)

### Schema update for delivery_request

Update schema for delivery_request and spawned deliveries

**Changes:**

- `delivery_request` schema changes:
  - startDate and endDate in ms
  - new field for spawned child deliveries
  - add `umbrella_id` to ref path
  - change name from `requestDuration` to `requestEndCriteria`
- `delivery` schema changes:
  - startTime, endTime in milliseconds
  - added field for parent `delivery_request`
  
**Tests:**
- Create a new recurring delivery request using UI
- Try both "recurring" and "end on <date>" repeat options
- Verify that UI on confirmation page looks ok
- View created `delivery_request` in firebase, confirm that the request lives under its umbrella org and that the `startDate` and `endDate` fields are in milliseconds. (https://www.epochconverter.com/)
